### PR TITLE
fix(saving): saving loader alignment

### PR DESCRIPTION
**Pull request by anamikaanu96**
Contributes to #3617 

{{Fix alignment of inline loading in save pattern state}}

#### What did you change? 
Added a style in packages/ibm-products-styles/src/components/Saving/_saving.scss 
min-block-size was changed to auto from 2rem

### CODE REVIEW

File: packages/ibm-products-styles/src/components/Saving/_saving.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
 @use '../../global/styles/project-settings' as c4p-settings;
 
 $block-class: #{c4p-settings.$pkg-prefix}--saving;
+// $carbon-prefix: 'cds';
```
Comment (elycheea) [inline packages/ibm-products-styles/src/components/Saving/_saving.scss:15]: Looks like you figured out you could use the `carbon-prefix` from settings. :)
We can remove this line now.

**Review by elycheea — COMMENTED** [review]
Nice! Just need to remove comments we no longer need and to run `yarn format` to address any spacing / line length issues. 🎉

**Comment by anamikaanu96** [comment]
I have read the DCO document and I hereby sign the DCO.

**Review by davidmenendez — CHANGES_REQUESTED** [review]
looks good. just need to remove line 15

**Review by elycheea — APPROVED** [review]

**Pull request merged**